docs/api/platform/Error.md

```diff
@@ -298,12 +331,101 @@ void save_all_errors() {
     mbed_clear_all_errors();
 }
 ```
+
+<a name="crash-reporting-api"></a>
+#### Using `mbed_get_reboot_error_info()` to retrieve the reboot error info
+In the example below, a status variable reboot_error_detected is set to 1 when the callback is invoked then inside the main() 
+function we read the reboot error info using mbed_get_reboot_error_info().
+
+```CPP
+mbed_error_ctx error_ctx;
+int reboot_error_detected = 0;
+
+//Callback during reboot
+void mbed_error_reboot_callback(mbed_error_ctx *error_context) 
+{
+    printf("error callback received");
+    reboot_error_detected = 1;
+}
+
+// main() runs in its own thread in the OS
+int main() 
+{
+    if (reboot_error_detected == 1) {
+        if (MBED_SUCCESS == mbed_get_reboot_error_info(&error_ctx)) {
+            printf("\nSuccessfully read error context\n");
+    }
+    //main continues...
+}
+```
+
+#### Using `mbed_get_reboot_fault_context()` to retrieve the fault context info
+The example code below checks for exception error(MBED_ERROR_HARDFAULT_EXCEPTION) using error_status in the error context 
+and then retrieves the fault context using mbed_get_reboot_fault_context().
+
+```CPP
+mbed_error_ctx error_ctx;
+mbed_fault_context_t fault_ctx;
+int reboot_error_detected = 0;
+
+//Callback during reboot
+void mbed_error_reboot_callback(mbed_error_ctx * error_context) 
+{
+    printf("error callback received");
+    reboot_error_detected = 1;
+}
+
+// main() runs in its own thread in the OS
+int main() 
+{
+    if (reboot_error_detected == 1) {
+        if (MBED_SUCCESS == mbed_get_reboot_error_info(&error_ctx)) {
+            printf("\nRead in reboot info\n");
+            if (error_ctx.error_status == MBED_ERROR_HARDFAULT_EXCEPTION) {
+               if (MBED_SUCCESS == mbed_get_reboot_fault_context(&fault_ctx)) {
+                   printf("\nRead in fault context info\n");
+               }
+            }
+        }
+    }
+    //main continues...
+}
+```
+
+#### Using `mbed_reset_reboot_error_info()` to clear the reboot error info
+`mbed_reset_reboot_error_info()` API can be used to clear the reboot error info if required by the application.
```

Comment:
Same thing as before in reference to the API reference. In my mind 
> ... using the Mbed OS Crash Reboot API 

is correct,

> ...using the mbed_reset_reboot_error_info() API

should be rephrased from API to function as a _member_ of the API.

Thoughts? I could be off the mark on this one.